### ISSUE TITLE
feature does not work with forEach method

### ISSUE THREAD

**Issue by JoxGarcia** [annaet/cypress-tags#11]
First, I really like this feature i was reading alot of how to add tags to cypress tests and this repo has resolved my problems but I found something that I dont know if its an issue or bug.

In my currently test-suite I have some object arrays that contains info and have something like:

```javascript
 let keys = Object.keys(object);
    keys.forEach((ob) => 

      it(`example', () => {
        ...
      });
    });
```

The feature, doesnt recognize this types of its.

I dont know if this is going to be fixed in the future, or im doing something wrong. 

Thanks.

**Comment by annaet** [comment]
Hi @JoxGarcia

Thanks for reporting this, and for providing an example! It sounds like a bug as it isn't something I'm currently testing. I'll look at adding it to my tests and will try to get a fix out for you.

**Comment by annaet** [comment]
Hi @JoxGarcia

I've just released `v0.0.19` which I think should fix your problem.

I believe the issue wasn't the `forEach` loops themselves, but instead the use of template expressions in the test title.

For example,
```
it(`I am a regular test ${foo}`, () => { });
```

I have added tests using `forEach` loops as well, just in case, but they appear to be working fine for me!

I'll close this issue for now, but please let me know if this didn't fix your problem.
